requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Testing and deployment packages.
-coverage==4.4.2
```

Comment:
Ideally, all this should go into a `pyproject.yaml`, but I wanted to keep the changes small.